something.yaml

```diff
@@ -0,0 +1,3 @@
+|
```

Comment:
Looks like this got left over. 🙂

Reply:
Whoops! was testing what a duration was serialized to